src/createFileTransform.ts

```diff
@@ -30,6 +30,23 @@ class Transformer extends Minipass {
   }
 }
 
+// Files and directories that have `_` as their first character in React Native CLI templates
+// These should be transformed to have `.` as the first character
+const UNDERSCORED_DOTFILES = [
```

Comment:
Do we have any file in the template that starts with an `_`? If not, we could just iterate all template files, and just rename them automatically (as opposed to hardcoding that list)?

Reply:
We do not have such files, but the template inside React Native repo does, as does `react-native-template-typescript`. This structure directly matches the structure used in community CLI, so that we convert the same files.

Reply:
Agreed @EvanBacon I'd prefer to only match the specific files called out in `UNDERSCORED_DOTFILES` so that our behavior matches the community CLI for this feature.

Reply:
Makes sense, just asking because these lists could get out of date. But the tradeoffs seems worse for not listing it 😄